README.md

```diff
@@ -57,16 +48,43 @@ $ export KEY=value
 $ export USER=someuser
 ```
 
-To set environment variables in Windows from CMD use the following command:
+To set environment variables in Windows from CMD:
 
 ```bash
 $ set KEY=value
 
 # For example
 $ set USER=someuser
 ```
+The below instructions for Docker and Python assume that you have already set `USER` and `PASSWORD` in your current environment.
+
+### Docker
+Follow the [official documentation](https://docs.docker.com/get-docker/) for installing Docker. You can then run a Docker container for the API in two ways:
+#### Option 1: Pull the latest image from Docker Hub
+```bash
+docker pull neurobagel/api
+docker run --name api -p 8000:8000 --env USER --env PASSWORD neurobagel/api
+```
+#### Option 2: Build the image using the Dockerfile
+After cloning the current repository, build the Docker image locally:
+```bash
+docker build -t neurobagel/api .
+docker run --name api -p 8000:8000 --env USER --env PASSWORD neurobagel/api
```

Comment:
Probably better to run detached (`-d`) so you don't start out attached and accidentally kill it etc. If you want, you could reference the docker docs so folks can refresh their memory on how to attach or see the logs.